src/workflow_validator.py

```
@@ -0,0 +1,22 @@
+import yaml
+
+
+class ValidationResult:
+    def __init__(self, is_valid, errors, workflow_data):
```

Ruff (0.14.6)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)